frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/ModelManager.java

```diff
@@ -116,6 +122,7 @@ public boolean unregisterModel(String modelName) {
         model.setMaxWorkers(0);
         wlm.modelChanged(model);
         model.getModelArchive().clean();
+        startupModels.remove(modelName);
```

Comment:
You don't have to remove it.

Reply:
We do. This can be deleted and it can come back through register API. We need to clean up.